prow/repoowners/repoowners.go

```diff
@@ -279,7 +279,7 @@ func (o *RepoOwners) walkFunc(path string, info os.FileInfo, err error) error {
 
 	// '.md' files may contain assignees at the top of the file in a yaml header
 	// Note that these assignees only apply to the file itself.
-	if o.enableMDYAML && filename != ownersFileName && strings.HasSuffix(filename, "md") {
+	if o.enableMDYAML && strings.HasSuffix(filename, ".md") {
```

Comment:
Why do we want to support this? Use an OWNERS file.

Reply:
This is pattern is used extensively throughout the k/website repo. See this [query](https://github.com/kubernetes/website/search?utf8=%E2%9C%93&q=approvers+filename%3A*.md&type=). 
We support this pattern because OWNERS files didn't previously support per-file `approvers` and `reviewers` as is now achievable with regexp filtering. By allowing OWNERS config in markdown headers, each file can have its own `approvers` and `reviewers` without having to be put in its own folder with its own OWNERS file.